DEFAULT_CONFIG.json5

```diff
@@ -284,6 +284,8 @@
         /// Therefore, the maximum batch size is 2^16-1 (i.e. 65535).
         /// The default batch size value is the maximum batch size: 65535.
         batch_size: 65535,
+        /// Perform automatic batching of messages if they are smaller of the batch_size
+        automatic_batching: true,
```

Comment:
`automatic_batching` can make one think a manual batching is possible if not automatic.
Why not just `batching: true` or `enable_batching: true` ?

Reply:
I'm fine with `batching`. I'll update the PR.